docs/application.html

```diff
@@ -228,13 +228,14 @@ <h1>XtkSchemaNode</h1>
   <tbody>
     <tr><td><b>children</b></td><td>A array/map of children of the node. See note on the ArrayMap structure below</td></tr>
     <tr><td><b>dataPolicy</b></td><td>Returns a string of characters which provides the data policy of the current node.</td></tr>
+    <tr><td><b>dbEnum</b></td><td>Returns a string of characters which provides the data policy of the current node.</td></tr>
     <tr><td><b>description</b></td><td>A long, human readable, description of the node</td></tr>
     <tr><td><b>descriptionLocalizationId</b></td><td>The translation id of the description of the node.</td></tr>
     <tr><td><b>editType</b></td><td>Returns a string of characters which specifies the editing type of the current node.
     <tr><td><b>enum</b></td><td>The name of the enumeration for the node, or an empty string if the node does node have an enumeration. See <b>enumeration()</b> method to get the corresponding <b>XtkSchemaNode</b></td></tr>
     <tr><td><b>enumerationImage</b></td><td>Returns the name of the image of the current node in the form of a string of characters.</td></tr>
     <tr><td><b>folderModel</b></td><td>Only on the root node, returns a string which contains the folder template(s). On the other nodes, it returns undefined.
-    <tr><td><b>image** </td><td>Returns the name of the image in the form of a string of characters.
+    <tr><td><b>image**</b></td><td>Returns the name of the image in the form of a string of characters.
```

Comment:
Why this line has changed?

Reply:
The <b> tag was not closed before (</b> was missing).